src/main/java/com/conveyal/datatools/manager/controllers/api/DeploymentController.java

```diff
@@ -173,6 +169,28 @@ private static FileInputStream downloadDeployment (Request req, Response res) th
         return fis;
     }
 
+    /**
+     * Export all the GIS shapefiles for the deployment.
+     */
+    private static String downloadGIS (Request req, Response res) throws IOException {
+        String type = req.queryParams("type");
+        Auth0UserProfile userProfile = req.attribute("user");
+        Deployment deployment = getDeploymentWithPermissions(req, res);
+
+        GisExportJob.ExportType exportType = GisExportJob.ExportType.valueOf(type);
+        File temp = File.createTempFile(type + "_" + deployment.name + "_" + type, ".zip");
+
+        DeploymentGisExportJob gisExportJob = new DeploymentGisExportJob(exportType, deployment, temp, userProfile);
+        JobUtils.heavyExecutor.execute(gisExportJob);
+
+        // Do not use S3 to store the file, which should only be stored ephemerally (until requesting
+        // user has downloaded file).
+        FeedDownloadToken token = new FeedDownloadToken(gisExportJob);
+        Persistence.tokens.create(token);
```

Comment:
What are we creating the token for? It's never used anywhere else (although this may just be my datatools ignorance)

Reply:
A similar token is created in the base GisExportJob, which is why I kept this in here. I think we may query these tokens later.